hazelcast.go

```diff
@@ -87,13 +87,13 @@ type Instance interface {
 	// Shutdown shuts down this Instance.
 	Shutdown()
 
-	// GetCluster returns the Cluster this instance is part of.
-	// Cluster interface allows you to add listener for membership
+	// GetClusterService returns the ClusterService this instance is part of.
+	// ClusterService interface allows you to add listener for membership
 	// events and learn more about the cluster this Hazelcast
 	// instance is part of.
-	GetCluster() core.Cluster
+	GetClusterService() core.ClusterService
```

Comment:
This naming is not consistent with hazelcast server. I think this should stay as Cluster.

Reply:
Right, is there a reason why it is called `Cluster` but not `ClusterService` on server side?

Reply:
I don't know. There is a ClusterService but it is in an internal package. Public API seems to be named as Cluster.